docs/guides/run-jobs-batch.mdx

```diff
@@ -23,12 +23,104 @@ Batch mode can shorten processing time if all jobs can be provided at the outset
 
 ![This diagram illustrates jobs submitted in a batch.  It shows five jobs, numbered 0 through 4, in a queue. The jobs are a mix of Estimator and Sampler.](/images/guides/batch.png 'Figure 1: Batch execution')
 
-There are multiple ways you can reconfigure your jobs to take advantage of the parallel processing provided by batching.
+## Open a batch
+
+You can open a runtime batch by using the context manager `with Batch(...)` or by initializing the `Batch`
+class. The batch starts when its first job begins execution.
+
+
+**Batch class**
+
+``` python
+from qiskit_ibm_runtime import Batch, Samplerv2 as Sampler, Estimatorv2 as Estimator
+
+backend = service.least_busy(operational=True, simulator=False)
+batch = Batch(service=service, backend=backend)
```

Comment:
We are also deprecating the `service` parameter, since we can infer the service if `backend` is an object.